pyparsing/diagram/__init__.py

```diff
@@ -1,5 +1,5 @@
 # mypy: ignore-errors
-from __future__ import annotations
+# from __future__ import annotations
```

Comment:
why not remove it?

Reply:
I've upgraded cython version to 3.0.11, and now its working